src/test/resources/edu/cornell/kfs/rass/rass-agency-basic-test.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Kfs>
+    <Awards/>
```

Comment:
there is no unit test that has both numeric and alphanumeric agency numbers. I wonder if we should add one such test or update the existing files?

Reply:
looks good. Formatting is a bit off in this file. Could you please re-format it?